sktime/forecasting/tests/test_predict_quantiles.py

```diff
@@ -0,0 +1,83 @@
+import pandas as pd
```

Comment:
ok, but this file applies to `AutoETS` only. Hence, put it into `test_ets` or `test_autoets` or a similar file.